src/QZXing-components.pri

```diff
@@ -450,6 +450,10 @@ unix {
 
     DEFINES += NOFMAXL
 
+    contains( CONFIG, ubports) {
```

Comment:
Why `ubports` here? Also, if this is needed on Ubuntu Touch, would it not also be needed on Android and other Halium-based systems?

Reply:
The only qmake consumer I found doing some way of CONFIG matching here was Nymea, and they specifically use `ubports` as their config.

Regarding the actual implications of the define, my guess is that the other platforms don't rotate the viewfinder to portrait like that. If I understood the code correctly the way the pixels are read they might be squished, not keeping aspect ratio intact. But as the maintainer mentioned in the comment above the define, fixing this properly would be preferred.

Reply:
Read my comment again, not to confuse anybody: not consumer of QZXing but any consumer that differentiates multiplatform builds using `ubports: {  }` in qmake.

Reply:
Yeah, I think it would be better to drop the extra config option and define for this, and fix it properly (though I'm not sure what a proper fix would be).

Reply:
When trying to support Sailfish OS, I introduced this config/define because I could not find a way to properly identify the platform. Thus, I will accept it as well for Ubuntu Touch since it will not affect other users.
Thank you @fredldotme (and @dobey for the conversation)